src/beast/beast/module/core/maths/Muldiv.cpp

```diff
@@ -0,0 +1,61 @@
+//------------------------------------------------------------------------------
+/*
+    This file is part of Beast: https://github.com/vinniefalco/Beast
+    Copyright 2013, Vinnie Falco <vinnie.falco@gmail.com>
+
+    Portions of this file are from JUCE.
```

Comment:
Header was copied and pasted from another file without checking. My bad. I will move `mulDiv` to `STAmount.h`